### DISCUSSION TITLE
Upgrading to latest version

### DISCUSSION THREAD

**Discussion by mcodetrail** [abraunegg/onedrive#1436] [Q&A] [answered]
It would be nice to have documentation about upgrating onedrive to latest release version.
Should I re-clone it and re-compile it again? Or maybe using 'apt' ?

**Comment by abraunegg** [comment, marked as answer]
@mbaldassarri 
There is documentation - please read https://github.com/abraunegg/onedrive/blob/master/docs/INSTALL.md

**Reply by ParamjitKahai** [reply]
I am sorry but I do not see upgrade instructions at the link you provided.  Are you saying that one should follow the installation instructions there, then?

**Reply by abraunegg** [reply]
@ParamjitKahai 
> I am sorry but I do not see upgrade instructions at the link you provided. Are you saying that one should follow the installation instructions there, then?

Correct - to upgrade you must re-install by following the installation instructions for your OS.

**Reply by ParamjitKahai** [reply]
Thank you much.
